### ISSUE TITLE
Shorter python import

### ISSUE THREAD

**Issue by sverhoeven** [eWaterCycle/remotebmi#31]
Now you have to do

```python
from remotebmi.client.client import RemoteBmiClient
```

it would nice if this was shorter

```python
from remotebmi import RemoteBmiClient
```

Same goes for the container clients and server main function (which might need to be renamed to more a library friendly name)

**Comment by BSchilperoort** [comment]
Yeah this would be much nicer! :+1:

**Comment by BSchilperoort** [comment]
Fixed this in #37, except for;

> server main function (which might need to be renamed to more a library friendly name
